README.adoc

```diff
@@ -1527,6 +1527,34 @@ else
 end
 ----
 
+=== `unless` with Multiple Conditions
+
+Do not use `unless` with multiple conditions.
+
+[source,ruby]
+----
+# bad
+unless sweet?(fruit) && !fruit.green?
+  yield fruit # skip yielding sweet non-green fruits
+end
+
+# bad
+return unless node.ternary? && !infinite_loop? && offense?(node)
+
+# good - non-inversed
+if !sweet?(fruit) || fruit.green?
```

Comment:
In the same vein, I don't have an impression that `if` reads so well with one of the conditions negated. "If not sweet or green" sounds ambiguous. "If green or not sweet" is slightly better.